src/app/plan/duck/sagas.ts

```diff
@@ -693,6 +719,274 @@ function* migrationPoll(action) {
   }
 }
 
+/******************************************************************** */
+//Hooks sagas
+/******************************************************************** */
+
+function* fetchHooksSaga(action) {
```

Comment:
This looks good to me, but I need to learn more about sagas and come back to look at all of these.